ewcfg.py

```diff
@@ -11773,6 +12587,198 @@ def atf_hooves(ctn=None):
 		},
 		products = ['slimepoudrin']
 	),
+EwSmeltingRecipe(
+		id_recipe = "humancorpse",
+		str_name = "a corpse",
+		alias = [
+			"stiff",
+			"corpse",
+			"deadperson",
+			"cadaver",
+		],
+		ingredients = {
+			'scalp': 20,
+			'dinoslimemeat':2,
+			'string':2
+		},
+		products = ['humancorpse']
+	),
+EwSmeltingRecipe(
+		id_recipe = "popeonarope",
+		str_name = "pope on a rope",
+		alias = [
+			"pope",
+			"francis",
+			"deadpope",
+			"sacrilege",
+		],
+		ingredients = {
+			'humancorpse': 1,
+			'diadem':1,
+			'scarf':1,
+			'confessionbooth':1
+		},
+		products = ['popeonarope']
+	),
+EwSmeltingRecipe(
+		id_recipe = "reanimatedcorpse",
+		str_name = "reanimated corpse",
+		alias = [
+			"frankenstein",
+			"reanimate",
+			"revenant",
+		],
+		ingredients = {
+			'humancorpse': 1,
+			'poudrin':10,
```

Comment:
aww, it doesn't actually use a soul?

Reply:
that's what i wanted to do at first. the issue was that you can't remove the soul and have it keep its user-bound properties. you just destroy it forever. i suppose you could do it theoretically but i'd rather keep souls in play if i can help it

Reply:
oh, it's already possible to permanently destroy souls at the recycling plant. it's very funny

Reply:
hmmm. yeah, just using poudrins feels pretty half assed. we can leave it as is for now but i'll change it in a patch later. i'm going to see if i can have my cake and eat it too.

Reply:
alright. but if we make it so it takes souls to craft in the future, the item should have different flavor text to differentiate it from the cheap poudrin variant

Reply:
you know, i don't like the sound of that. i'll fix it before the patch drops instead.

Reply:
just updated the code.